### PR TITLE
feat(toggle): add 'size' attribute

### PR DESCRIPTION
**Pull request by jadekler**
Note: this removes 3 toggle related variables from pui-variables.scss

[Finishes #134017213]

### CODE REVIEW

**Comment by jadekler** [comment]
@charleshansen Thoughts on removing the following?

```
$toggle-bg-width
$toggle-bg-height
$toggle-diameter
```

We felt they were somewhat useless now that there are three possible sizes; we could certainly backfill `toggle-bg-small-width`, `toggle-bg-medium-width`, etc. etc., but thought that might have been overkill, so decided to just do away with them alltogether.

**Comment by jadekler** [comment]
@charleshansen One last note - we were hoping for final styling check by Justin

File: library/spec/pivotal-ui-react/toggle/toggle_spec.js [hunk 1]
```diff
@@ -70,4 +70,35 @@ describe('Toggle', () => {
...
   });
+
+  describe('size attribute', function() {
+    beforeEach(function() {
+      ReactDOM.unmountComponentAtNode(root);
+    });
+
+    it('renders with size=medium by default', function() {
+      ReactDOM.render(<Toggle id='boggle' />, root);
+      expect($('label').attr('class')).toContain('medium');
```
Comment (charleshansen) [inline library/spec/pivotal-ui-react/toggle/toggle_spec.js]: I see why you didn't want to use jquery here. There is a pattern you can use

```
let subject;
beforeEach(() => {
  subject = ReactDOM.render(<Toggle id="boggle"/>, root);
})

it('renders with medium', () => {
 expect(...)
})

it('renders with small', () => {
  subject::setProps({size: 'small'});
 expect(..).toContain('small');
})
```

File: library/src/pivotal-ui/components/forms/forms.scss [hunk 2]
```diff
@@ -285,9 +285,48 @@ This form looks and behaves mostly like a content editable field
     cursor: pointer;
     outline: none;
     text-indent: -99999px;
-    width: $toggle-bg-width;
-    height: $toggle-bg-height;
-    border-radius: $toggle-bg-height;
+
```
Comment (charleshansen) [inline library/src/pivotal-ui/components/forms/forms.scss]: If at all possible, do not include actual pixel values in files other than pui-variables.scss, this makes maintenance much easier.

I'm hoping for `$toggle-bg-width-sm;`, `$toggle-bg-width-md`, etc. There will be some pixel values left over, hopefully those can be calculated, see below...

File: library/src/pivotal-ui/components/forms/forms.scss [hunk 3]
```diff
@@ -285,9 +285,48 @@ This form looks and behaves mostly like a content editable field
...
+    &.small {
+      height: 14px;
+      width: 26px;
+      border-radius: 14px;
+
+      &:after {
+        height: 12px;
+        width: 12px;
+
```
Comment (charleshansen) [inline library/src/pivotal-ui/components/forms/forms.scss]: for example, the `12px` here can be `$toggle-bg-height-sm - 2px`

File: library/spec/pivotal-ui-react/toggle/toggle_spec.js (hunk 1 above)
Comment (jadekler) [inline library/spec/pivotal-ui-react/toggle/toggle_spec.js]: @charleshansen Is this something you feel strong about, or only mild? I think it's somewhat easier to read the test with mounting directly about the expectation (I don't have to jump around trying to find out what subject is, and so on). Thoughts?

File: library/src/pivotal-ui/components/forms/forms.scss (hunk 2 above)
Comment (jadekler) [inline library/src/pivotal-ui/components/forms/forms.scss]: @charleshansen 👍 a19c22dab44ee86a57ffb03dec7814b112358ac8 thoughts on naming?

Also, what was the `bg` acronym?

File: library/src/pivotal-ui/components/forms/forms.scss (hunk 3 above)
Comment (jadekler) [inline library/src/pivotal-ui/components/forms/forms.scss]: @charleshansen 👍 a19c22dab44ee86a57ffb03dec7814b112358ac8

File: library/src/pivotal-ui/components/forms/forms.scss (hunk 2 above)
Comment (charleshansen) [inline library/src/pivotal-ui/components/forms/forms.scss]: Naming looks good. I'm assuming `bg` means background, but I think your names are a little more clear.

File: library/spec/pivotal-ui-react/toggle/toggle_spec.js (hunk 1 above)
Comment (charleshansen) [inline library/spec/pivotal-ui-react/toggle/toggle_spec.js]: I understand `subject` doesn't work for everyone, you don't need to use it. 

Looking at the original file, there are already some kinda weird testing patterns in there (namely some extra `unmountComponentAtNode`), but you didn't add, so I'll deal with those separately.

**Pull request merged**